### ISSUE TITLE
Search bar leads to datasektionen.se

### ISSUE THREAD

**Issue by Benjaneb** [datasektionen/styrdokument-bawang#6]
If you search for something the results will wrongly link to `datasektionen.se` instead of `styrdokument.datasektionen.se`.

**Comment by Herkarl** [comment]
The [fuzzyfile](https://taitan-styrdokument.datasektionen.se/fuzzyfile) seems to link to `datasektionen.se`. I'm guessing that the url might be hardcoded in [taitan](https://github.com/datasektionen/taitan).

**Comment by Herkarl** [comment]
Yup, here's the culprit: https://github.com/datasektionen/taitan/blob/0d634298f2a9f819942a0215aa2c9ee7669b2d23/fuzz/fuzz.go#L38
